hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -201,7 +201,7 @@
   // generated: A module is templated and generated as part of topgen
   // top_only: A module is not templated but is specific to 'top_*' instead of 'ip'
   module: [
-    { name: "uart0",    // instance name
+    { name: "uart",     // instance name
```

Comment:
are we backing out the multiple uart commit also?

Reply:
Not sure yet, reverting only the first patch didn't make a difference to the FPGA test, so there's something odd going on.